src/content/go/gorrila_mux/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "gorrila/mux"
+date: 2022-12-02T13:33:00+05:30
+draft: false
+weight: 1
+tags: ["go", "gorrila", "mux", "http"]
+---
+
+This library provides a middleware that extracts SQLCommenter HTTP request tags from a request being handled by [`gorrila/mux`](https://github.com/gorilla/mux) and attaches them to the request's context. This same context, when used to run queries using [sqlcommenter/go/database/sql](/go/database_sql), allows request tags and traceparent(if using [otelmux](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/github.com/gorilla/mux/otelmux)) to be passed into SQL comments.
+
+## Installation
+
+```shell
+go get -u github.com/google/sqlcommenter/go/gorrila/mux
+```
+
+## Usage
+
+```go
+import (
+    "net/http"
+
+    sqlcommentermux "github.com/google/sqlcommenter/go/gorrila/mux"
+    "github.com/gorilla/mux"
+)
+
+func runApp() {
+    r := mux.NewRouter()
+    r.Use(sqlcommentermux.SQLCommenterMiddleware)
+
```

Comment:
we can remove the empty lines in embedded code to make the vertical length more compact.

Reply:
I think vertical lines help with readability in docs too